toml/src/main/kotlin/org/rust/toml/intentions/ExtractDependencySpecificationIntention.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.toml.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiParserFacade
+import com.intellij.psi.util.parentOfType
+import org.rust.cargo.CargoConstants
+import org.rust.lang.core.psi.ext.endOffset
+import org.rust.toml.isDependencyListHeader
+import org.toml.lang.psi.TomlInlineTable
+import org.toml.lang.psi.TomlKeyValue
+import org.toml.lang.psi.TomlPsiFactory
+import org.toml.lang.psi.TomlTable
+
+class ExtractDependencySpecificationIntention : TomlElementBaseIntentionAction<TomlKeyValue>() {
+    override fun getText() = "Extract dependency in separate table"
```

Comment:
This differs from `"Extracts dependency specification to a separate table in Cargo.toml"` (`description.html`)

WDYT about `"Extract dependency specification in Cargo.toml"` for consistency with `ExpandDependencySpecificationIntention`?

Reply:
Then it is better to be `Extract dependency specification`, similar to `ExpandDependencySpecificationIntention` and to not repeat that it is `Cargo.toml` intention in `Cargo.toml` file.

<img width="500" src="https://user-images.githubusercontent.com/6342851/115709279-b51ed880-a379-11eb-991d-16c95b7613dc.png">